betago/model.py

```diff
@@ -57,7 +65,7 @@ def next_move():
             json_result = jsonify(**result)
             return json_result
 
-        self.app.run(host='0.0.0.0', debug=True, threaded=True, use_reloader=False)
+        self.app.run(host='0.0.0.0', port=int("8081"), debug=True, threaded=True, use_reloader=False)
```

Comment:
I could use `os.getenv("PORT")` (which is actually `8080` in c9) but I don't know if that is a universal enviroment variable or not.